workspaces/liferay-dlfoldertemplate-workspace/client-extensions/liferay-dlfoldertemplate-custom-element/src/components/template-list/TemplateList.js

```diff
@@ -304,8 +304,8 @@ const TemplateList = () => {
 				<ClayEmptyState
 					description={null}
 					imgProps={{alt: 'Alternative Text', title: 'Hello World!'}}
-					imgSrc={`${Liferay.ThemeDisplay.getPathThemeImages()}/states/search_state.gif`}
-					imgSrcReducedMotion={`${Liferay.ThemeDisplay.getPathThemeImages()}/states/search_state_reduced_motion.gif`}
+					imgSrc={`${Liferay.ThemeDisplay.getPathThemeImages()}/states/search_state.svg`}
```

Comment:
Let's not make changes to files in workspaces since we don't know which environment they are expected to be deployed to.

Reply:
Done!